contributing.md

```diff
@@ -48,7 +48,8 @@ Anyone can view and comment on active changes here.
 Reviewing others' changes is a good way to learn how the change process works and gain exposure 
 to activity in various parts of the code. You can help by reviewing the changes and asking 
 questions or pointing out issues -- as simple as typos or small issues of style.
-See also https://spark-prs.appspot.com/ for a convenient way to view and filter open PRs.
+See also <a href="https://spark-prs.appspot.com/">https://spark-prs.appspot.com/</a> for a
```

Comment:
@EnricoMi, can you generate HTML too accordingly? See https://github.com/apache/spark-website#generating-the-website-html

Reply:
I did, I run jekyll 4.0.0 and it generated a lot of unrelated changes. I will try again with <4.0.0.

Reply:
Yeah, it happens due to different versions. You can manually exclude unrelated changes except this line

Reply:
Done